packages/dashboard/src/components/lists/Watchlist.js

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { tradeActions } from '../../redux/actions'
+
+import {
+  Alert,
+  Container,
+  Row,
+  Col,
+  InputGroup,
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem,
+  Card,
+  CardHeader,
+  CardBody,
+} from 'shards-react'
+
+
```

Comment:
dont worry about formatting now, I'll run that script after. But deffs look into vscode auto formatting so the formatting diffs dont dirty up PRS